client-management-service-impl/src/main/java/io/mosip/esignet/services/ClientManagementServiceImpl.java

```diff
@@ -173,4 +198,60 @@ public io.mosip.esignet.core.dto.ClientDetail getClientDetails(String clientId)
         }
         return dto;
     }
+
+    @CacheEvict(value = Constants.CLIENT_DETAIL_CACHE, key = "#clientDetailCreateV2Request.getClientId()")
+    @Override
+    public ClientDetailResponse createOIDCClientV2(ClientDetailCreateV2Request clientDetailCreateV2Request) throws EsignetException {
+        Optional<ClientDetail> result = clientDetailRepository.findById(clientDetailCreateV2Request.getClientId());
+        if (result.isPresent()) {
+            log.error("Duplicate Client Id : {}", ErrorConstants.DUPLICATE_CLIENT_ID);
+            throw new EsignetException(ErrorConstants.DUPLICATE_CLIENT_ID);
+        }
+
+        ClientDetail clientDetail = toClientDetailDTO(clientDetailCreateV2Request);
+
+        String clientName = toClientName(
```

Comment:
change toClientName to getClientNameLanguageMapAsJsonString

Reply:
please check this comment [5a0b048](https://github.com/mosip/esignet/pull/265/commits/5a0b04816d36a5791956c438e0181a232c3f231a)